packages/pageheader/components/PageHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+import { BreadcrumbProps } from "../../breadcrumb/components/Breadcrumb";
+
+export interface PageHeaderProps {
+  children: {
+    title: React.ReactNode;
+    breadcrumb: React.ReactElement<BreadcrumbProps>;
+    actions?: React.ReactNode[];
+  };
+}
+
+class PageHeader extends React.PureComponent<PageHeaderProps, {}> {
+  public render() {
+    const { title, breadcrumb, actions = [] } = this.props.children;
+
+    return (
+      <div>
+        <h1>{title}</h1>
+        {breadcrumb}
+        <ul>
+          {actions.map((action, index) => (
+            <li key={`action${index}`}>{action}</li>
```

Comment:
Indexes are unsuitable as react keys. If an action is added and an action is deleted in the same update, React would render the wrong thing because the key is not unique to any particular action

Reply:
I feared that, what would you propose as a better key?

Reply:
You'd need to delegate this to callers...

`actions={[{ actionId: "myaction1", action: <Clickable /> }]}`

Reply:
Another option is to use a global counter so you always get a unique key, but this blows the whole point of keys and forces a dom change on update.